<launch> 
    <!-- Blood Oxigenation Measurement Sensor -->    
    <node name="g3t1_1" pkg="component" type="g3t1_1" output="screen" />

    <param name="frequency" value="0.2" /> <!-- 0.2 Hz  -->

    <!-- Defines the percentages to consider low, moderate or high risk -->
    <param name="lowrisk" value="0,20" />
    <param name="midrisk" value="21,65" />
    <param name="highrisk" value="66,100" />

    <!-- accuracy in percentage -->
    <param name="accuracy" value="90" type="double" />

</launch>